src/main/fc/rc.c

```diff
@@ -416,19 +442,18 @@ static FAST_CODE void processRcSmoothingFilter(void)
             const bool ready = (currentTimeMs > 1000) && (targetPidLooptime > 0);
             if (ready) { // skip during FC initialization
                 // Wait 1000ms after power to let the PID loop stabilize before starting average frame rate calculation
-                if (rxIsReceivingSignal() && isRxIntervalValid) {
+                if (rxIsReceivingSignal() && isRxRateValid) {
```

Comment:
maybe:
```suggestion
                if (rxIsReceivingSignal() && getRxRateValid()) {
```

Reply:
@ledvinap - Could you explain  the benefit in making this change?

`getRxRateValid()` just returns the value of the static variable `isRxRateValid`, which is directly available in rc.c to the `processRcSmoothingFilter` function.

Avoiding the function call to evaluate `getRxRateValid()` seemed more efficient, that's all.  it can still be called by external files when needed, but locally, I wasn't sure why we would run it within this file.

Reply:
Its about consistency - use only one method to access rxRateValid information.
Compiler (with LTO for use from another file) is smart enough to avoid(inline) function call , so there shouldn't be any performance penalty.